### PR TITLE
Corrects linting

### PR DESCRIPTION
**Pull request by dcalvoalonso**
Currently, the code does not pass the linting check. I have fixed this issue and included `npm run lint` as part of Travis.

### CODE REVIEW

File: lib/plugins/attributeAlias.js [hunk 1]
```diff
@@ -27,10 +27,6 @@
 
 var config = require('../commonConfig'),
     utils = require('./pluginUtils'),
-    logger = require('logops'),
```
Comment (fgalan) [inline lib/plugins/attributeAlias.js]: Not sure if remove this is a good idea... I understand the reason is hat *currently* is an unused variable but I think is a good idea to have the logger object initiated (in partciular, the op string associated to each module) in the case logger needs to be used *in the future*.

Could a `/* jshin ... */` expection be used for this?

Comment (dcalvoalonso) [reply lib/plugins/attributeAlias.js]: Fixed with 749e1751e41ea144efdd10e6ef2f45a4f73a00bd and f467b5aaa51fdb4391d84664ff2129d92599dcba

File: lib/services/groups/groupService.js [hunk 2]
```diff
@@ -172,20 +172,27 @@ function remove(service, subservice, resource, apikey, device, callback) {
...
                 } else {
                     if (devices && devices.count > 0){
-                      for(var device of devices.devices) {
-                        deviceService.unregister(device.id, service, subservice, function(error) {
+
+                        async.map(devices.devices, unregisterDevice, function(error) {
                             if (error) {
                                 callback(error);
-                            }
+                            }
```
Comment (fgalan) [inline lib/services/groups/groupService.js]: Trailing whitespace should be removed?

Comment (dcalvoalonso) [reply lib/services/groups/groupService.js]: Fixed with 082030fb26574c2bb4f1992ced8cec24cd8d5e12

**Comment by fgalan** [comment]
The PR is ready for merge. However, I'd suggest to delay the merge of the PR until issue https://github.com/telefonicaid/iotagent-node-lib/issues/711 gets fixed (in order to add more entropy to master branch while the issue is investigated). It is fine with you @dcalvoalonso ?

**Comment by dcalvoalonso** [comment]
No problem at all!

File: .jshintrc [hunk 3]
```diff
@@ -17,6 +17,7 @@
   "node": true,
   "expr": true,
   "unused": "vars",
+  "esversion": 6,
```
Comment (fgalan) [inline .jshintrc]: PR already got LGT but I have realized about something...

What about test/.jshintrc file? Should it be modified the same way?

Comment (dcalvoalonso) [reply .jshintrc]: Well, it is not really needed since ECMAScript 6 syntax is not used in the tests. Nevertheless, it does not hurt to add it. :)

Comment (dcalvoalonso) [reply .jshintrc]: Fixed with 563d687b2b11004109478e9d3ea1a9fd084d7ded

**Comment by fgalan** [comment]
It seems there are some small conflicts in this PR that needs to be solved before being able to merge.

**Comment by dcalvoalonso** [comment]
Conflicts solved with 63a2d9ccbe9d107c8d4975cea19a0f33511093a5

**Comment by fgalan** [comment]
Merging. Thanks!

Do you plan to do a similar PR to add lint step to travis (and eventualy fix the code) for the other repos (IOTA-UL, IOTA-JSON, IOTA-LWM2M and LWM2M lib)?

**Comment by dcalvoalonso** [comment]
I have just submitted new PRs for IOTA-U and IOTA-JSON.  For IOTA-LWM2M and LWM2M lib, this was already done in previous releases.

**Pull request merged**